energy.md

```diff
@@ -249,20 +255,16 @@ The wind and solar technologies are electrolysis technologies, but are specifica
 
 ### Buildings
 
-* Structure and functional forms
+GCAM disaggregates the building sector into residential and commercial and models three aggregate services (heating, cooling, and other). Within each region, each type of building and each service starts with a different mix of fuels supplying energy (see Figure below). The future evolution of building energy use is shaped by changes in (1) floorspace, (2) the level of building service per unit of floorspace, and (3) fuel and technology choices by consumers.  Floorspace depends on population, income, and exogenously specified satiation levels. The level of building service demands per unit of floorspace depend on climate, building shell conductivity, income, and satiation levels.
 
-*Under Construction*
-
-### Industry
+<img src="gcam-figs/building_energy_2010.png" width="750" height="450" />
 
-* Cement
+Per-capita Residential and Commercial Energy Use in 2010
```

Comment:
Is this meant to be a caption for the figure on line 260?  If so, the blank line in between them will likely cause the formatting to be messed up (also, it doesn't have the figure caption style marker).